common/ast/ast_test.go

```diff
@@ -72,8 +73,30 @@ func TestASTCopy(t *testing.T) {
 		if err != nil {
 			t.Errorf("ast.ToAST() failed: %v", err)
 		}
-		if !reflect.DeepEqual(checked, checkedRoundtrip) {
-			t.Errorf("Roundtrip got %v, wanted %v", checkedRoundtrip, checked)
+		opts := []cmp.Option{
```

Comment:
Which would you prefer? The change is smaller, but the failure output is largely meaningless in the current code (a set of pointers) while in the change the actually differences are shown. If that's not important for you, then the smaller change would win, otherwise the options are needed for the diff rendering.

Reply:
I think it would be preferable to reduce the dependency set. I know the errors will not be as useful when there are issues; however, I also don't expect to be modifying these files much except for bug fixes, so any regression should standout right away.

Reply:
Thank you, Dan!